src/assets/scss/app.scss

```diff
@@ -64,7 +64,12 @@
 @include motion-ui-animations;
 
 $message-new: $white;
+$message-new-text: #404040;
 $message: $body-background;
+$message-text: darken($message, 20);
+
+$font-sizes: (1rem, 0.9rem, 0.85rem);
```

Comment:
nebylo by logičtější využít na tyto fonty  mapu ? A návázat na klíče breakpointu frameworku, aby se to celé chovalo co konzistentněji ? Tedy
 $font-sizes: (small: rem-calc(13), ... )

Reply:
krásně budeš moci pak i na to udělat breaky 
@include breakpoint (medium up)  ...

Reply:
prace s textem prozatim odebrana, nez prostuduji rem-calc